go-controller/pkg/cluster/gateway_init.go

```diff
@@ -83,6 +83,11 @@ func generateGatewayNATRules(ifname string, ip string) []iptRule {
 		args: []string{"-o", ifname, "-m", "conntrack", "--ctstate",
 			"RELATED,ESTABLISHED", "-j", "ACCEPT"},
 	})
+	rules = append(rules, iptRule{
+		table: "filter",
+		chain: "OS_FIREWALL_ALLOW",
```

Comment:
What is this "OS_FIREWALL_ALLOW"? It cannot be a standard?

Reply:
@pecameron We should not hard code the name of the chain. Get it from an 'env var' maybe? And default to INPUT. And put the env in daemonset configmap if at all

```
chain := os.GetEnv("FIREWALL_ALLOW_CHAIN")
if len(chain) == 0 {
     chain = "INPUT"
}
```

Reply:
@rajatchopra @shettyg This is 3.11 only. When @dcbw get the revised gateway implementation this will change. Inserting at the top of INPUT works well. I just put it at the head of INPUT. The env seems excessive.

Reply:
@pecameron For non-openshift deployments, FIREWALL_ALLOW_CHAIN would fail, no? Bcause, there may be no such chain or it may mean something else. Is FIREWALL_ALLOW_CHAIN some kind of standard?

Reply:
Wait, FIREWALL_ALLOW_CHAIN is just the env name of the actual chain name. If its not defined, then INPUT should be used, thats all.
I almost agree with Phil here that the env maybe excessive actually. So just inserting at INPUT HEAD might be good enough.

Reply:
@shettyg you are right. I changed it to inset into INPUT, which is standard. I think (but may be wrong), FIREWALL_ALLOW_CHAIN comes from firewalld.